pkg/k8sclient/k8s_utils.go

```diff
@@ -91,19 +91,19 @@ func GetMachineNetworkCIDR(configMap *v1.ConfigMap, log logrus.FieldLogger) (str
 }
 
 func GetSSHPublicKey(configMap *v1.ConfigMap, log logrus.FieldLogger) (string, error) {
```

Comment:
not sure why make format did this.
looking into it.

Reply:
ok, this is actually ok.
It was added [here](https://github.com/openshift/assisted-service/pull/757/files#diff-76f163ff886da1c6aaa9a9696fdd4ebe3cb4e97a32b14c5415b79e064f4aaddcR93) with excessive indentation